Code/Tools/AssetProcessor/native/AssetManager/assetProcessorManager.cpp

```diff
@@ -2886,6 +2886,19 @@ namespace AssetProcessor
 
     void AssetProcessorManager::AssessFilesFromScanner(QSet<AssetFileInfo> filePaths)
     {
+        if (m_initialScanSkippingFeature)
+        {
+            for (const AssetFileInfo& fileInfo : filePaths)
+                AddKnownFoldersRecursivelyForFile(fileInfo.m_filePath, fileInfo.m_scanFolder->ScanPath());
```

Comment:
The coding standard calls for brackets for all control statements
```suggestion
            for (const AssetFileInfo& fileInfo : filePaths)
            {
                AddKnownFoldersRecursivelyForFile(fileInfo.m_filePath, fileInfo.m_scanFolder->ScanPath());
            }
```